.vscode/settings.json

```diff
@@ -136,5 +136,8 @@
   },
 
   // Pin CI for the current branch to the status bar.
-  "github-actions.workflows.pinned.workflows": [".github/workflows/ci.yaml"]
+  "github-actions.workflows.pinned.workflows": [
+    ".github/workflows/ci.yaml"
+  ],
+  "C_Cpp.errorSquiggles": "enabled"
```

Comment:
What's this?

Reply:
I accidentally disabled errorSquiggles so i turned them back on. I thought it was a local setting